helpers/cuentasContablesHelper/cuentasContables.helper.go

```diff
@@ -172,10 +173,14 @@ func AsientoContable(totales map[int]float64, tipomvto string, descripcionMovto
 	transaccion.Etiquetas = string(jsonString)
 	transaccion.FechaTransaccion = time_bogota.Tiempo_bogota()
 
+	tercerodebito := true
+	tercerocredito := true
 	for clave, _ := range totales {
+		logs.Debug("******** La clave ", clave)
 		urlcuentas := "http://" + beego.AppConfig.String("catalogoElementosService") + "cuentas_subgrupo/?query=SubgrupoId.Id:" + strconv.Itoa(clave) + ",Activo:true,SubtipoMovimientoId:" + tipomvto
 		logs.Info(urlcuentas)
 		if respuesta, err := request.GetJsonTest(urlcuentas, &elemento); err == nil && respuesta.StatusCode == 200 {
+			logs.Debug("***** encontro la cuenta")
```

Comment:
Comentar
```suggestion
			// logs.Debug("***** encontro la cuenta")
```

o Eliminar
```suggestion
```